core/model/modx/processors/element/propertyset/updatefromelement.class.php

```diff
@@ -1,4 +1,5 @@
 <?php
+// https://github.com/modxcms/revolution/issues/12580
```

Comment:
@OptimusCrime I did not say anything against this comment. http://joxi.ru/v29QeZnHozVq2G